com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/ProtoBuilder.java

```diff
@@ -159,7 +196,13 @@ public void build(Task task) throws CompileExceptionError,
     @Override
     public void clearState() {
         super.clearState();
-        protoParams = null;
         srcBuilders = null;
     }
+
+    // Update digest with the signature of the output proto format
+    @Override
+    public void signature(MessageDigest digest) {
```

Comment:
we do call this function for each task, so I just use it to include output protobuf message digest into the task digest